examples/pipelines/simple_pipeline/components/image_cropping/fondant_component.yaml

```diff
@@ -0,0 +1,27 @@
+name: Image cropping
+description: Component that removes single-colored borders around images and crops them appropriately
+image: ghcr.io/ml6team/image_cropping:latest
+
+input_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+
+output_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+      width:
+        type: int16
+      height:
+        type: int16
+
+args:
+  cropping_threshold:
+    description: Threshold parameter used for detecting borders. A lower (negative) parameter results in a more performant border detection, but can cause overcropping. Default is -30
+    type: int
+  padding:
+    description: Padding for the image cropping
```

Comment:
Could you also clarify how padding changes the algorithm's behaviour in the description?